release/README.md

```diff
@@ -37,7 +37,15 @@
   ```bash
   ./earthly --build-arg RELEASE_TAG --push ./release+release-homebrew
   ```
-* Important: Subscribe to the PR that griswoldthecat created in homebrew-core, so that you can address any review comments that may come up.
+* Visit the newly created pull request at `https://github.com/earthly/homebrew-earthly/pull/XXX`; there should be two checks that are automatically applied:
```

Comment:
Maybe say to look in the `#release` internal Slack channel and click on the homebrew-earthly PR link?

Reply:
sounds good, I changed it to that as well as point out that there's a link in the previous commands stdout.